sdk/maintenance/Microsoft.Azure.Management.Maintenance/src/Generated/Models/MaintenanceConfiguration.cs

```diff
@@ -84,9 +85,10 @@ public MaintenanceConfiguration()
         /// day23,day24, recurEvery: Month Last Sunday, recurEvery: Month
         /// Fourth Monday.</param>
         /// <param name="visibility">Gets or sets the visibility of the
-        /// configuration. Possible values include: 'Custom', 'Public'</param>
-        public MaintenanceConfiguration(string id = default(string), string name = default(string), string type = default(string), string location = default(string), IDictionary<string, string> tags = default(IDictionary<string, string>), string namespaceProperty = default(string), IDictionary<string, string> extensionProperties = default(IDictionary<string, string>), string maintenanceScope = default(string), string startDateTime = default(string), string expirationDateTime = default(string), string duration = default(string), string timeZone = default(string), string recurEvery = default(string), string visibility = default(string))
-            : base(id, name, type)
+        /// configuration. The default value is 'Custom'. Possible values
+        /// include: 'Custom', 'Public'</param>
+        public MaintenanceConfiguration(string id = default(string), string name = default(string), string type = default(string), SystemData systemData = default(SystemData), string location = default(string), IDictionary<string, string> tags = default(IDictionary<string, string>), string namespaceProperty = default(string), IDictionary<string, string> extensionProperties = default(IDictionary<string, string>), string maintenanceScope = default(string), string startDateTime = default(string), string expirationDateTime = default(string), string duration = default(string), string timeZone = default(string), string recurEvery = default(string), string visibility = default(string))
```

Comment:
Is there any way SystemData could we moved to the end of the parameter list (this happens in multiple functions)? Otherwise PR looks good to me.

Reply:
Code is autogenerated we cant touch it otherwise pipeline will break.